examples/greeter-extension/src/component.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-greeter-extension
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {createBindingFromClass} from '@loopback/context';
+import {Component} from '@loopback/core';
+import {ChineseGreeter} from './greeters/greeter-cn';
+import {EnglishGreeter} from './greeters/greeter-en';
+import {GreeterExtensionPoint} from './greeter-extension-point';
+
+export class GreeterComponent implements Component {
+  bindings = [
+    createBindingFromClass(GreeterExtensionPoint, {
+      key: 'greeter-extension-point',
+    }),
+    createBindingFromClass(EnglishGreeter, {namespace: 'greeters'}),
+    createBindingFromClass(ChineseGreeter, {namespace: 'greeters'}),
```

Comment:
Can we use `asGreeter` template please?

Reply:
I mean why the built-in greeters are created with `{namespace: 'greeters'}` but without applying the changes made by `asGreeter` and why `asGreeter` does not set any namespace?